src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      
+      <div class="stars stars--1">
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      
+      <div class="stars stars--2">
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star stars__star--active"></div>
```

Comment:
Still not fixed from the previous review
<img width="784" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/2c501d9b-89e5-46ff-94b3-2a8d678b6d0b">
```suggestion
        <div class="stars__star"></div>
        <div class="stars__star"></div>
```